statsrelay.go

```diff
@@ -29,6 +29,12 @@ const BUFFERSIZE int = 1 * 1024 * 1024 // 1MiB
 // Such as <prefix>.statsProcessed.  Default is "statsrelay"
 var prefix string
 
+// prefix is the string that will be prefixed onto each metric passed
```

Comment:
I'd love it to make these comply with Go's comment guidelines.

https://golang.org/doc/effective_go.html#commentary

Specifically, so that comments start with the name of the variable/function they describe.